docs/create_usb_boot_disks.rst

```diff
@@ -0,0 +1,214 @@
+Create USB Boot Disks
+=====================
+
+Overview
+-------------
+For the initial SecureDrop setup, you will need three USB disks.
+
+Two of them will will be used for the Tails operating system,
+for creating the *Admin Workstation* and *Secure Viewing Station*.
+
+The other USB disk will have the Ubuntu Server installer,
+which is needed to install the underlying server OS for your
+*application* and *monitor* servers.
+
+Although the Ubuntu Server disk will be used later in the setup process,
+we recommend downloading and creating the disk at the same time you create
+the Tails USB disks to avoid having to repeat steps, and to avoid slower
+download speeds later that would be introduced by trying to download Ubuntu
+over the TOR network from the *Admin Workstation.*
```

Comment:
I think you can save this entire paragraph; you're already explaining what the purpose of the Ubuntu disk is in the preceding paragraph, and we can reasonably assume that most folks will just follow the instructions in the order in which we present them.